core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2311,6 +2311,249 @@ secret_configs:
         namespace: test-credentials
         name: registry-pull-credentials
         type: kubernetes.io/dockerconfigjson
+  # This secret contains the pull secrets for all possible registries.
+  - from:
+      .dockerconfigjson:
+        dockerconfigJSON:
```

Comment:
how are we going to keep all of those in sync?

Reply:
done